Create ui-c8y-1024-10-4-small-min-max-datapoint-ranges-are-now-correctly-shown-on-the.md (#4982)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1024-10-4-small-min-max-datapoint-ranges-are-now-correctly-shown-on-the.md

---
date: ""
title: Small min-max data point ranges now display correctly on chart Y-axis
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67256
version: 1024.10.4
---

> We improved how chart Y-axes render when handling tight value ranges. Instead of repeating identical numbers on the axis, charts now automatically adjust tick intervals to show accurate intermediate decimal values. This makes tracking subtle data variations significantly easier and prevents distorted chart visuals.

Reviewer: ```suggestion
The rendering of the chart Y-axis when handling tight value ranges has been improved. Instead of repeating identical numbers on the axis, charts now automatically adjust tick intervals to show accurate intermediate decimal values. This makes tracking subtle data variations significantly easier and prevents distorted chart visuals.
```
